tests/test_main.py

```diff
@@ -22,6 +22,7 @@
                      get_species_by_label,
```

Comment:
Please change the commit message: `Tests: Modifications to... to account for...`